pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1073,7 +1080,11 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 					prefix := ""
 					// no error check, we assume that CNI type was set correctly
 					if net.Multus != nil {
-						prefix = "net"
+						if net.Multus.Default {
```

Comment:
(I am not exactly sure about it but) Now that we use JSON format to pass networks to CNI, and the JSON dicts contain `name` attributes which list interface names, do we really need to handle `eth` interface prefix here? (This is not a request to change, just a question. If we don't need it, we can clean up in a follow-up.)

Reply:
Can you point to commit/pr where this change was made?

Reply:
I see it now.

Reply:
@greg-bock here https://github.com/kubevirt/kubevirt/pull/2107 it was merged a day ago.